docs/root/operations/admin.rst

```diff
@@ -174,3 +174,21 @@ The fields are:
   .. http:get:: /runtime?format=json
 
   Outputs /runtime in JSON format. This can be used for programmatic access of runtime values.
+
+.. http:get:: /hystrix_event_stream
+
+  Start streaming :ref:`event stream<https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events>` to hystrix dashboard. 
+  
+  The dashboard should set the stream source to this admin endpoint.
+
+Note on usage of Hystrix dashboard to visualize Envoy statistics: 
```

Comment:
Can you verify this looks as expected in the doc output? You will need to fix the docs build.